src/client/opamClient.ml

```diff
@@ -1158,6 +1158,7 @@ let remove_t ?ask ~autoremove ~force atoms t =
     let to_keep =
       (if autoremove then t.installed_roots %% t.installed else t.installed)
       -- to_remove -- full_orphans -- orphan_versions
+      ++ universe.u_base (* includes base packages of the universe *)
     in
```

Comment:
I believe `universe.u_base` should be on the left of the `--`: otherwise a user specifying a base package to be removed could be silently ignored ?

Also... tautological comment ? ;)